Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,275 @@
+            .cornerRadius(20)
+            .padding()
+
+            .onAppear {
+                Purchases.configureInCustomEntitlementsComputationMode(apiKey: Constants.apiKey,
+                                                                       appUserID: appUserID)
+
+                Task {
+                    self.offerings = try await Purchases.shared.offerings()
+                    print("offerings: \(String(describing: offerings))")
+                }
+            }
+
+            NavigationView {
+                List(customerInfoStreamFires) { customerInfoStreamFire in
+                    NavigationLink(destination: CustomerInfoDetailsView(customerInfo: customerInfoStreamFire.customerInfo)) {
+
+                        VStack (alignment: .leading) {
+                            HStack {
+                                Text("Fired at:")
+                                    .bold()
+                                Text("\(customerInfoStreamFire.date, formatter: dateFormatter)")
+                            }
+                            HStack {
+                                Text("App User ID:")
+                                    .bold()
+                                Text(customerInfoStreamFire.customerInfo.originalAppUserId)
+                            }
+                        }
+                    }                }
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .principal) {
+                        VStack {
+                            Text("CustomerInfo stream values")
+                                .font(.headline)
+                                .foregroundColor(.primary)
+                                .padding()
+                            Text("List fills in below when new values arrive.")
+                                .foregroundColor(.primary)
+                        }
+                    }
+                }.onAppear {
+                    subscribeToCustomerInfoStream()
+                }
+            }
+        }
+        .sheet(isPresented: $showingLogInAlert) {
+            CustomAlert(inputText: $appUserID, logIn: logIn)
+        }
+        .sheet(isPresented: $showingExplanation) {
+            ExplanationView()
+        }
+    }
+
+    func logIn(_ appUserID: String) {
+        print("Switching to user: \(appUserID)")
+        Task {
+            try await Purchases.shared.logIn(appUserID)
+        }
+    }
+
+    func subscribeToCustomerInfoStream() {
+        self.streamTask = Task {
+            for await customerInfo in Purchases.shared.customerInfoStream {
+                DispatchQueue.main.async {
+                    print("got new customerInfo: \(customerInfo)")
+                    customerInfoStreamFires.append(CustomerInfoStreamFires(customerInfo: customerInfo, date: Date()))
+                }
+            }
+        }
+    }
+
+    func purchaseFirstOffering() async throws {
```

Comment:
I think you meant to make this not `throws` since you're logging the error inside.

Reply:
good catch, I added the catch halfway through and forgot to update the signature